examples/nfs-volume-clone/README.md

```diff
@@ -0,0 +1,156 @@
+    zfssa-nfs-volume-example-pvc   Bound    pvc-c7ac4970-8ae1-4dc8-ba7f-0e37a35fb39d   50Gi       RWX            zfssa-nfs-volume-clone-example-sc   7s
+    ```
+3. Display the pod mounting the volume
+
+    The command `kubectl get pods` should now return something similar to this:
+    ```text
+    NAME                           READY   STATUS    RESTARTS   AGE
+    zfssa-csi-nodeplugin-ph8qr     2/2     Running   0          8m32s
+    zfssa-csi-nodeplugin-wzgpq     2/2     Running   0          8m32s
+    zfssa-csi-provisioner-0        4/4     Running   0          8m32s
+    zfssa-nfs-volume-example-pod   1/1     Running   0          2m21s
+    ```
+
+## Write data to the volume
+
+Once the pod is deployed, write data to the volume in the pod:
+```yaml
+kubectl exec -it pod/zfssa-nfs-volume-example-pod -- /bin/sh
+/ # cd /mnt
+/mnt # ls
+/mnt # echo "hello world" > demo.txt
+/mnt # 
+```
+
+Write as much data to the volume as you would like.
+
+## Remove the pod
+
+For this step, do *not* use `helm uninstall` yet. Instead, delete the
+example pod: `kubectl delete pod/zfssa-nfs-volume-example-pod`.
+
+This leaves the PVC intact and bound to the share on the
+Oracle ZFS Storage Appliance.
+
+## Deploy pods using clones of the original volume
+
+```
+helm  install -f local-values/local-values.yaml clone-volume ./clone-volume
+```
+
+Once deployed, there should be two new PVCs (clones of the original) and two
+new pods with the cloned volume mounted. Without writing, the cloned filesystems
+will have the same data as the original filesystem. The filesystems are different
+and will diverge with writes.
+
+2. Display the volumes
+   The command `kubectl get pvc` should now return something similar to this:
+    ```text
+    NAME                STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS       AGE
+    zfssa-csi-nfs-pvc   Bound    pvc-808d9bd7-cbb0-47a7-b400-b144248f1818   10Gi       RWX            zfssa-csi-nfs-sc   8s
+zfssa-nfs-volume-clone-pvc-0
+zfssa-nfs-volume-clone-pvc-1
+
+    ```
+3. Display the pods mounting the clones
+
+   The command `kubectl get pods` should now return something similar to this:
+    ```text
+    NAME                             READY   STATUS    RESTARTS   AGE
+    pod/zfssa-csi-nodeplugin-lpts9   2/2     Running   0          25m
+    pod/zfssa-csi-nodeplugin-vdb44   2/2     Running   0          25m
+    pod/zfssa-csi-provisioner-0      2/2     Running   0          23m
+zfssa-nfs-volume-clone-pod-0
+zfssa-nfs-volume-clone-pod-1
+
+    ```
+
+## Verify the data to the volumes
+
+Once the pod is deployed, write data to the volume in the pod:
+```yaml
+kubectl exec -it zfssa-nfs-volume-clone-pod-0 -- /bin/sh
+/ # cd /mnt
+/mnt # ls
+/mnt # echo "hello world" > demo.txt
+/mnt # 
+```
+
+## Remove the example when complete
+
+Helm will remove all of our pods and data:
+```
+helm uninstall setup-volume
```

Comment:
Shouldn't the order matter? I thought if you delete the original volume, all snapshots & clones from the volume are also deleted. If they are not deletable, deletion fails.

Reply:
I'm going to remove clone from this branch, the branch will be version update and bug fixes only, I will address your comments in the clone branch when I re-create it. Thanks Jeongtae!